src/nidcpower/examples/nidcpower_advanced_sequence.py

```diff
@@ -33,28 +33,33 @@ def example(resource_name, channels, options, voltage_max, current_max, points_p
 
         with session.initiate():
             session.wait_for_event(nidcpower.Event.SEQUENCE_ENGINE_DONE)
-            measurements = session.fetch_multiple(points_per_output_function * 2, timeout=timeout)
+            channels = resource_name.split(',')
+            measurement_group = [None] * len(channels)
+            for i, channel in enumerate(channels):
+                channel_name = channel.strip()
+                measurement_group[i] = session.channels[channel_name].fetch_multiple(points_per_output_function * 2, timeout=timeout)
```

Comment:
Suggestion:

Instead of
```
            channels = resource_name.split(',')
            measurement_group = [None] * len(channels)
            for i, channel in enumerate(channels):
                channel_name = channel.strip()
                measurement_group[i] = session.channels[channel_name].fetch_multiple(points_per_output_function * 2, timeout=timeout)
```

You can do it this way and be more pythonic:
```         channels = resource_name.split(',')
            channels = [x.strip() for x in resource_name.split(',')]
            measurement_group = [session.channels[channel_name].fetch_multiple(points_per_output_function * 2, timeout=timeout) for channel_name in channels]
```

Reply:
Good idea, thanks.